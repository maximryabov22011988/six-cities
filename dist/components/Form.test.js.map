{"version":3,"file":"Form.test.js","sourceRoot":"","sources":["../../src/views/components/Form.test.tsx"],"names":[],"mappings":";;AAAA,+BAA+B;AAC/B,gDAAgD;AAEhD,iCAA0B;AAE1B,EAAE,CAAC,uBAAuB,EAAE,GAAG,EAAE;IAC/B,MAAM,IAAI,GAAG,QAAQ;SAClB,MAAM,CACL,oBAAC,cAAI,IACH,SAAS,EAAC,aAAa,EACvB,MAAM,EAAC,KAAK,EACZ,QAAQ,EAAE,IAAI,CAAC,EAAE,EAAE,IAElB,IAAI,CACA,CACR,CAAC,MAAM,EAAE,CAAC;IAEb,MAAM,CAAC,IAAI,CAAC,CAAC,eAAe,EAAE,CAAC;AACjC,CAAC,CAAC,CAAC"}