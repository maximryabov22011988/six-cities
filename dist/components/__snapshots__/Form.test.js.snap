// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form correctly render 1`] = `
<form
  className="login__form form"
  method="get"
  onSubmit={[MockFunction]}
/>
`;
